src/hubinit/hub.bal

```diff
@@ -65,17 +66,29 @@ public function main() {
     }
     );
     if (result is websub:Hub) {
+
         webSubHub = result;
-        if (unsentMessages.length() > 0) {
-            foreach var unsentMessage in unsentMessages {
-                var publishResponse = webSubHub.publishUpdate(unsentMessage.topic, unsentMessage.message.toBytes());
-                if (publishResponse is error) {
-                    log:printError("Error notifying hub: " +
-                        <string>publishResponse.detail()?.message);
-                } else {
-                    log:printInfo("Update notification successful!");
-                }
-                runtime:sleep(2000);
+        if (config:getAsBoolean("mosip.hub.republish_enable", false)) {
+            task:AppointmentData appointmentData = {
+                seconds: config:getAsString("mosip.hub.republish-job-seconds"),
```

Comment:
can we change the below config to mosip.hub.cron-task.unsent-messages-republish.seconds pattern ?

Reply:
done!

Reply:
ok resolved